include/unifex/unstoppable.hpp

```diff
@@ -22,16 +22,54 @@
 
 namespace unifex {
 namespace _unstoppable {
+
+template <typename Sender>
+struct _sender final {
+  struct type;
+};
+
+template <typename Sender>
+using sender = typename _sender<remove_cvref_t<Sender>>::type;
+
+template <typename Sender>
+struct _sender<Sender>::type final {
+  UNIFEX_NO_UNIQUE_ADDRESS Sender sender_;
+
+  template <
+      template <typename...>
+      class Variant,
+      template <typename...>
+      class Tuple>
+  using value_types = sender_value_types_t<Sender, Variant, Tuple>;
+
+  template <template <typename...> class Variant>
+  using error_types = sender_error_types_t<Sender, Variant>;
+
+  static constexpr bool sends_done = sender_traits<Sender>::sends_done;
+
+  template <typename Receiver>
+  friend auto tag_invoke(tag_t<connect>, type self, Receiver&& r) noexcept(
+      is_nothrow_connectable_v<Sender, remove_cvref_t<Receiver>>) {
+    return connect(
+        with_query_value(
+            std::move(self).sender_, get_stop_token, unstoppable_token{}),
+        static_cast<Receiver&&>(r));
+  }
+};
```

Comment:
You should probably add an implementation of `blocking`.
```suggestion
  friend auto tag_invoke(tag_t<blocking>, const type& s) noexcept {
    return blocking(s.sender_);
  }
};
```